api/operator/v1beta1/vmextra_types.go

```diff
@@ -130,20 +131,34 @@ func isContainsFinalizer(src []string) bool {
 
 // RemoveFinalizer - removes vm-operator finalizer from finalizers list.
 // executes provided callback if finalizer found
-func RemoveFinalizer(src client.Object, andThen func(client.Object) error) error {
+func RemoveFinalizer(src client.Object, keepOwnerReference bool, andThen func(client.Object) error) error {
```

Comment:
It's better to create new function, that will patch remove owner reference. 

* it will reduce amount of changes
* it will not increase code complexity
* we need this function only at 1 place

Reply:
done